docs/notebooks/survival_model.ipynb

```diff
@@ -0,0 +1,1587 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    S0 = sp.stats.expon.sf
</pre><p>Could you add a little about the <code>sf</code> function? From SciPy docs it says it's the survival function. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/543/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>